docs/vision/strategy-2030.md

```diff
@@ -9,156 +9,139 @@ slug: /vision/strategy-2030
-| **Real World Assets (RWA)** | Tokenize traditionally illiquid assets (real estate, fine art) to increase liquidity, accessibility, and operational efficiency via smart contracts. | *Tokenomicon* (refinement of tokenomics for stability and growth). |
-| **Supply Chain Traceability** | Provide a tamper-proof ledger from origin to consumer, leveraging transaction metadata to enhance transparency, prevent fraud, and facilitate targeted recalls. | *Global Identity* (linking physical assets to verifiable digital identities). |
-| **Global Payments** | Establish fast, secure, and low-cost cross-border payments utilizing Layer 2 scaling (Hydra) and native assets to bypass traditional intermediaries. | *The Internet Hydra-ted* (L2 scalability for high-volume transactions). |
+### F.1. Scalability & Interoperability
 
-**Key Performance Indicators (KPIs)**
+| Focus Area | Description | Expected Enhancement |
+| :---- | :---- | :---- |
+| **L1 protocol improvements** | Improve consensus, ledger, and networking to raise throughput and shorten finality. | L1 capacity for institutional, retail, and enterprise demand. |
+| **L2 integration** | Make L2 solutions first-class so high-volume activity can move off L1 and settle back. | High-frequency, low-latency transactions with L1 security. |
+| **Cross-chain interoperability** | Standardize secure bridges and state-proofs to other chains and legacy infra. | Cardano as an interoperability hub. |
+| **Core ZK capabilities** | Provide modular ZK infrastructure for private, verifiable off-chain computation. | Privacy-preserving, verifiable apps. |
 
-| Metric | Target (2030) | Rationale |
-| :--- | :--- | :--- |
-| **Annual L1 Revenue** | **$200** Million | The ultimate metric of economic self-sufficiency and utility. |
-| **Daily Revenue Transactions** | `> 250,000` transactions (non-ADA value) | Indicates diverse network utility (metadata, RWA policy IDs, identity) beyond simple ADA transfers. |
-| **Enterprise Integrations** | `> 15` Major corporations/financial institutions utilizing Cardano for core operations. | Measure success in translating high-assurance design into business adoption. |
-| **Developer Onboarding Time** | Median time for new developer to deploy first smart contract `< 48 hours` | Improve the developer experience and platform attractiveness. |
+### F.2. Security & Resilience
 
-### A.2. User & Business Experience
+| Focus Area | Description | Expected Enhancement |
+| :---- | :---- | :---- |
+| **Post-quantum readiness** | Migrate protocol-critical cryptography to PQ-resistant candidates. | Proactive protection against future threats. |
+| **Client diversity** | Support additional full-node and light-client implementations with conformance testing. | Higher resilience, better decentralization. |
```

Comment:
Client diversity has it's benefits, but resilience is not one of them. The risk of a single implementation behaving different from another one is much higher than a single implementation agreeing with itself across versions.